### DISCUSSION TITLE
Checkpoint for T5 trained on News data?

### DISCUSSION THREAD

**Discussion by apoorvumang** [google-research/text-to-text-transfer-transformer#777] [Q&A]
Hi

Is it possible to get access to T5 pretrained model (any size) that was trained on just news data?

(Reason: I am working on KG completion on Wikidata using T5 and getting some convincing results. However, T5 pretraining using C4 means it has read Wikipedia which can be a source of test leakage. I am looking for any pretrained T5 that does not have wikipedia in its pretraining dataset.)

It would be very helpful for my research.

Also, if it's not possible, could you suggest how I should go about training it myself?

Thanks

**Comment by craffel** [comment]
Hi, no such checkpoint is available, but you can follow these instructions to train from scratch:
https://github.com/google-research/text-to-text-transfer-transformer#training
You can modify this task:
https://github.com/google-research/text-to-text-transfer-transformer/blob/master/t5/data/tasks.py#L46
to use a different dataset. You could try RealNews
https://github.com/rowanz/grover/tree/master/realnews

**Reply by apoorvumang** [reply]
Thankyou!
